phpng_xhprof.c

```diff
@@ -146,10 +146,10 @@ typedef struct hp_entry_t
 } hp_entry_t;
 
 /* Various types for XHPROF callbacks       */
-typedef void (*hp_init_cb)(TSRMLS_D);
```

Comment:
In case you are wondering why all this was removed.

TSRMLS_DC/TSRMLS_CC/TSRMLS_D/TSRMLS_C are not longer required.